### DISCUSSION TITLE
mouse actions without SIP

### DISCUSSION THREAD

**Discussion by basbebe** [asmvik/yabai#947] [Q&A] [answered]
I'm trying out yabai 3.3.10 without disabling SIP.

I assumed that `mouse_follows_focus` and `focus_follows_mouse` wouldn't work but `mouse_modifier`, `mouse_action1` and `mouse_action2` would.

```
yabai -m config mouse_modifier               fn
yabai -m config mouse_action1                move
yabai -m config mouse_action2                resize
yabai -m config mouse_drop_action            swap
```

`mouse_drop_action` works.
I thought that I would just be able to click anywhere on the window while holding `fn` to move it but it behaves just like before yaobai was installed.

Am I getting something wrong? Does this only work with SIP disabled?

**Comment by basbebe** [comment, marked as answer]
I just found `move window by clicking anywhere in its frame` listed in https://github.com/koekeishiya/yabai/wiki/Disabling-System-Integrity-Protection so it seems like I had wrong expectations.

**Reply by asmvik** [reply]
You can toggle a macOS setting to allow this behaviour natively, but the trigger is not customizable. 
Run `defaults write -g NSWindowShouldDragOnGesture -bool true` and reboot. You can now move windows from anywhere by holding ctrl + cmd.
